src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -36,11 +54,47 @@ def create_not_exists_filter_group(term):
     return create_filter_group(filter_list, u"AND")
 
 
+def create_greater_than_filter_group(term, value):
+    filter_list = [create_query_filter(term, u"GREATER_THAN", value)]
+    return create_filter_group(filter_list, u"AND")
+
+
+def create_less_than_filter_group(term, value):
+    filter_list = [create_query_filter(term, u"LESS_THAN", value)]
+    return create_filter_group(filter_list, u"AND")
+
+
 class FileEventFilterStringField(QueryFilterStringField):
+    """Helper class for creating filters with the ``EXISTS``/``NOT_EXISTS`` filter clauses."""
+
     @classmethod
     def exists(cls):
+        """Returns a FilterGroup to find events where filter data exists."""
         return create_exists_filter_group(cls._term)
 
     @classmethod
     def not_exists(cls):
+        """Returns a FilterGroup to find events where filter data does not exist."""
         return create_not_exists_filter_group(cls._term)
+
+
+class FileEventFilterSizeField(object):
+    """Helper class for creating filters with the ``GREATER_THAN``/``LESS_THAN`` filter clauses."""
+
+    _term = u"override_boolean_field_name"
+
+    @classmethod
+    def greater_than(cls, value):
```

Comment:
Might be good to say Args: value (int) here and mention the unit... bytes?

Reply:
I see that it is in `FileSize`.. still might be worth putting here as well

Reply:
Nevermind, I see that it is in `FileSize` and I guess we could use this base class for other units

Reply:
Is `FileEventFilterComparableField` or something a tad more generic? We could add filters that are not sizes but still have ge and lt

Reply:
yeah, that makes sense to genericize it.